pages.es/windows/slmgr.md

```diff
@@ -0,0 +1,8 @@
+# slmgr
+
+> Este comando es un alias de `slmgr.vbs`.
+> Más información: <https://learn.microsoft.com/windows-server/get-started/activation-slmgr-vbs-options>.
+
+- Muestra la documentación del comando original:
```

Comment:
```suggestion
- Ver documentación para el comando original:
```

Reply:
I have unified with "Vea la documentación para el .  Thank you :)